packages/frontend/src/components/common/SpaceActionModal/CreateSpaceModalContent.tsx

```diff
@@ -38,26 +28,35 @@ const renderUser = (user: {
     firstName: string;
     lastName: string;
     role: string;
-}) => {
-    return (
-        <FlexWrapper key={user.userUuid}>
-            <UserCircle>
-                {user.firstName.substr(0, 1) + user.lastName.substr(0, 1)}
-            </UserCircle>
+}) => (
+    <Group spacing="sm" position="apart" noWrap>
+        <Group>
+            <Avatar radius="xl" tt="uppercase" color="blue">
```

Comment:
Are these looking blue at all to you? I see them as grey even though they are set to blue here.

Reply:
can you take a screenshot? there are few places where I had to update colors

Reply:
It seems like everywhere I see these avatars they are gray. Whatever their access.

<img width="617" alt="Screenshot 2023-12-19 at 16 02 02" src="https://github.com/lightdash/lightdash/assets/1864179/6c0e3a38-8cc2-48bc-8db2-c5c05adf005a">

Reply:
must be render not picking up the changes. triggering manual sync!